packages/build/src/publish.ts

```diff
@@ -24,6 +25,7 @@ export default async function publish(
 
   await githubRepo.promoteRelease(config);
 
+  publishNpmPackages();
```

Comment:
Maybe stupid question, but: right now, the `npm run evergreen-release` happens in its own task on evergreen. After that, a new task for the `release_publish` is trigger which checks out the code again, runs install and then triggers the publish.
At this point, the versions in the npm packages are fresh from checkout, i.e. they are the "dev" versions. We would have to trigger setting the version inside `publishNpmPackages()`, too, right? 🤔

Reply:
That is the reason why we have `bumpNpmPackages` outside the if in `release.ts`, essentially the bump is done again.